JS/Lesson22/sinan/index.html

```diff
@@ -1,11 +1,51 @@
 <!DOCTYPE html>
-<html lang="en">
```

Comment:
The lang attribute in the HTML tag is used to specify the language of the document. It's important for accessibility and SEO (Search Engine Optimization) purposes. There're several reasons why it's needed:

- Accessibility: Screen readers and other assistive technologies use the lang attribute to determine the language of the content. This allows them to provide appropriate pronunciation and language-specific behaviors to assist users with disabilities. It ensures that users who rely on these technologies can access and understand the content correctly.

- SEO: Search engines also use the lang attribute to understand the language of a web page. This helps search engines present relevant search results to users searching in a particular language. It can improve the discoverability of your content in search results for users searching in that language.

- Internationalization: If you have a multilingual website, specifying the language using lang can help browsers and search engines understand which language variant of a page to display to users based on their language preferences or location.